addons/web/static/src/js/views/basic/basic_controller.js

```diff
@@ -114,6 +114,14 @@ var BasicController = AbstractController.extend(FieldManagerMixin, {
         return this.mutex.exec(function () {})
             .then(this._discardChanges.bind(this, recordID || this.handle, options));
     },
+    _nextPage: function () {
```

Comment:
those functions are private, they should be in the Private section, ordered alphabetically and properly documented. See our new (saas-16) coding guidelines: https://github.com/odoo/odoo/wiki/Javascript-coding-guidelines